Dockerfile

```diff
@@ -19,8 +19,9 @@ RUN cargo build --release --bin atuin
 FROM debian:bullseye-20230320-slim AS runtime
 
 RUN useradd -c 'atuin user' atuin && mkdir /config && chown atuin:atuin /config
-RUN apt update && apt install ca-certificates -y # so that webhooks work
-WORKDIR app
+# Install ca-certificates for webhooks to work
+RUN apt update && apt install ca-certificates -y && rm -rf /var/lib/apt/lists/*
```

Comment:
any reason why not using `apt-get clean`?

Reply:
This are two different caches, AFAIKT: package list cache, created by `apt-get update` and packages cache, created by `apt-get install`.

And, as stated in [Docker Best Practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/), `apt-get clean` is running by default, so we only need to clean list cache here.

![Screenshot_20230510-093858](https://github.com/ellie/atuin/assets/829944/d55fe88f-ee7a-48e5-9a4b-2171e770cf82)